File: 2248-intersection-of-multiple-arrays/2248-intersection-of-multiple-arrays.cpp
class Solution {
public:
    vector<int> intersection(vector<vector<int>>& nums) {
        int map[1001] = {};
        for(vector<int>& arr : nums)
        {
            for(int& n : arr) map[n]++;
        }
        vector<int> v;
        int sz(nums.size());
        for(int i = 0; i < 1001; i++)
        {
            if(map[i] == sz) v.push_back(i);
        }
        return v;
    }
};

File: 2248-intersection-of-multiple-arrays/2248-intersection-of-multiple-arrays.py
class Solution:
    def intersection(self, nums: List[List[int]]) -> List[int]:
        mp = [0] * 1001
        for arr in nums:
            for n in arr:
                mp[n] += 1
        res = []
        for i in range(len(mp)):
            if mp[i] == len(nums):
                res.append(i)
        return res